### PR TITLE
Save column title after pagination

### LINKED ISSUE #5399: updateColumnTitle is undone after pagination
**Bootstraptable version(s) affected**:
1.18.0

**Description**  
When a column title is set with 'updateColumnTitle' it will be reset when pagination is used

**Example**  
https://live.bootstrap-table.com/code/dirkoppelt/5160

### PR DESCRIPTION
**Pull request by Moutard3**
**Bug fix?**
yes

**New Feature?**
no

**Resolve an issue?**
Fix #5399 

**Example(s)?**
Before: https://live.bootstrap-table.com/code/dirkoppelt/5160
After: https://live.bootstrap-table.com/code/Moutard3/5177 (moutard3:fix/5399)

### CODE REVIEW

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -3190,15 +3190,15 @@ class BootstrapTable {
       this.options.escape ? Utils.escapeHTML(params.title) : params.title
 
     if (this.columns[this.fieldsColumnsIndex[params.field]].visible) {
-      const header = this.options.height !== undefined ? this.$tableHeader : this.$header
-
-      header.find('th[data-field]').each((i, el) => {
+      this.$header.find('th[data-field]').each((i, el) => {
```
Comment (wenzhixin) [inline src/bootstrap-table.js:3193]: There's no need to modify it here.
Error: https://live.bootstrap-table.com/code/wenzhixin/5204

Comment (djhvscf) [inline src/bootstrap-table.js:3193]: What happen if the `data-height` is set?

**Review by djhvscf — CHANGES_REQUESTED** [review]

File: src/bootstrap-table.js (hunk 1 above)
Comment (wenzhixin) [reply src/bootstrap-table.js:3193]: I have updated to `this.resetView()` to solve the problem.

**Comment by wenzhixin** [comment]
After: https://live.bootstrap-table.com/code/wenzhixin/5204

**Review by wenzhixin — APPROVED** [review]

**Pull request merged**